lib/plan_picker/accounts/user.ex

```diff
@@ -0,0 +1,146 @@
+defmodule PlanPicker.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @derive {Inspect, except: [:password]}
+  schema "users" do
+    field :email, :string
+    field :password, :string, virtual: true
+    field :hashed_password, :string
+    field :confirmed_at, :naive_datetime
+
+    field :index_no, :string
+    field :last_name, :string
+    field :name, :string
```

Comment:
Added those fields to user model.

Reply:
Just a formality - wouldnt something like `:first_name` be more fitting here?
Don't know if such a small change is meaningful enough to go through the trouble of changing it, though.

Reply:
Doesn't really matter to me, you can change it if you want to.

Reply:
Hm, changing it might generate more problems than it would solve. Maybe it is better to leave it as-is.

Reply:
Sure.